stac_fastapi/pgstac/stac_fastapi/pgstac/extensions/query.py

```diff
@@ -0,0 +1,48 @@
+"""Pgstac query customisation."""
+
+import operator
+from enum import auto
+from types import DynamicClassAttribute
+from typing import Any, Callable, Dict, Optional
+
+from pydantic import BaseModel
+from stac_pydantic.utils import AutoValueEnum
+
+from stac_fastapi.extensions.core.query import QueryExtension as QueryExtensionBase
+
+
+class Operator(str, AutoValueEnum):
```

Comment:
I believe these operators find use in the `search` extension and not just `query`, so perhaps the previous spot (or a different file for operators) is more sensible than here

Reply:
@moradology What is the search extension. Not come across that one. Happy to move it for the purposes of merging the PR.

Reply:
As the query extension is deprecated in favour of filter extension and using something like [pygeofilter](https://github.com/geopython/pygeofilter) to parse the input for the filter extension will perform error checking I am not sure if this operator class has any use outside of the query extension. 

If not, I would say it should stay here so that it can be removed in a later release of stac-fastapi without worrying whether it is used by something else.